courses/machine_learning/deepdive2/structured/solutions/5a_train_keras_ai_platform_babyweight.ipynb

```diff
@@ -0,0 +1,1593 @@
+{
```

Comment:
<p>typo: train_examples=20000 not train_examples=2000000</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/674/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>Nah, I don't want the students actually running 2M examples, but when I did this is what I got. Otherwise it will take a LONG time to finish (i.e. after the class ends haha)</p>

Reply:
aha i see ok.